piff/input.py

```diff
@@ -254,6 +273,13 @@ def __init__(self, config, logger=None):
                             separately to each input catalog.)  [default: None]
             :nproc:         How many multiprocessing processes to use for reading in data from
                             multiple files at once. [default: 1]
+            :reserve_frac:  Reserve a fraction of the stars from the PSF calculations, so they
+                            can serve as fair points for diagnostic testing.  These stars will
+                            have outlier rejection performed along with the regular stars, but
```

Comment:
I'm confused.  I thought you said in the PR description that, by default, the reserve stars will _not_ have outlier rejection applied.

Reply:
Yep.  That's an error.  When I first wrote the code, I thought we'd want that.  But then I made it optional, and by default not.  I failed to update this doc string.